src/doc/dev_manual/MemoryLeaks.rst

```diff
@@ -49,7 +49,7 @@ Building VisIt_
 Just like the third party libraries, VisIt_ needs to be built with debug
 support in order for Valgrind to produce useful stack traces. Furthermore,
 VisIt_ contains conditional code that does additional cleanup at exit to
-eliminate spurious memory leaks that is enabled with VISIT_DEBUG_LEAKS.
+eliminate spurious memory leaks that is enabled with DEBUG_MEMORY_LEAKS.
```

Comment:
"that is" to "that are"

Reply:
I changed it a little differently than you suggested.